backend/config/database.js

```diff
@@ -1,23 +1,67 @@
 module.exports = {
-  telemetryDatabase: {
+  readOnlyIndexerDatabase: {
     dialect: "postgres",
-    host: process.env.NEAR_TELEMETRY_DATABASE_HOST,
-    database: process.env.NEAR_TELEMETRY_DATABASE_NAME,
-    username: process.env.NEAR_TELEMETRY_DATABASE_USERNAME,
-    password: process.env.NEAR_TELEMETRY_DATABASE_PASSWORD,
+    host:
```

Comment:
It's still pretty ambiguous for me - why the property is called readonly, while the env can be either readonly or not-readonly?
Should we call variable `NEAR_INDEXER_DATABASE_HOST` something like `NEAR_READ_WRITE_INDEXER_DATABASE_HOST`?

Reply:
Those are fallback arguments to keep the backwards compatibility

Reply:
> Should we call variable NEAR_INDEXER_DATABASE_HOST something like NEAR_READ_WRITE_INDEXER_DATABASE_HOST?

We don't write to it from Explorer side, so it is read-only.

For telemetry, we have a master database instance where we write the data when Explorer receives the telemetry requests, and then we also read the data from a read replica.